test/TestEnvVarsInCommand.py

```diff
@@ -1,9 +1,8 @@
 import unittest
+from test import RunFromText
```

Comment:
isort should be made aware of the first-party test helpers. It's best to use an explicit relative import for this.